cfn.properties

```diff
@@ -0,0 +1,21 @@
+name=Cloudformation Plugin for SonarQube
+category=External Analysers
+descripton=Cloudformation rules, supporting cfn-nag
+homepageUrl=https://github.com/Hack23/sonar-cloudformation-plugin
+archivedVersions=1.0.1
+publicVersions=1.0.2
+
+defaults.mavenGroupId=com.hack23.sonar
+defaults.mavenArtifactId=sonar-cloudformation-plugin
+
+1.0.2.description=Update dependency com.fasterxml.jackson.core:jackson-databind to v2.10.0.pr2
+1.0.2.date=2019-09-08
+1.0.2.sqVersions=[7.7,LATEST]
+1.0.2.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-1.0.2
+1.0.2.downloadUrl=https://oss.sonatype.org/service/local/repositories/releases/content/com/hack23/sonar/sonar-cloudformation-plugin/1.0.2/sonar-cloudformation-plugin-1.0.2.jar
+
+1.0.1.description=New rules added from cfn-nag
+1.0.1.date=2019-08-17
+1.0.1.sqVersions=[7.7,LATEST]
```

Comment:
If you are going to include this old version, you must update the compatibility range: only one version can be compatible with `LATEST`. The convention is to set the end of the range to the version of SQ that's current at the time of the release of the plugin replacement.